aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -547,7 +547,7 @@ std::optional<Tensor> convert_boolean_attn_mask(const std::optional<Tensor>& att
 // TODO(eqy): delete this when cuDNN -inf issue is resolved
 std::optional<Tensor> convert_boolean_attn_mask_cudnn(const std::optional<Tensor>& attn_mask, caffe2::TypeMeta dtype) {
   // TODO Use the max type of the input and output
-  return convert_boolean_attn_mask_(attn_mask, dtype, -65504.0);
+  return convert_boolean_attn_mask_(attn_mask, dtype, -1024.0);
```

Comment:
I dont love this numerical fiddiling tbh, seems brittle and may help in some cases but hurt in others. I think I could craft an adversarial example that exposes it in this case with such a low fill value. When I added the SafeSoftMax PR, I think the trick was to actually special case on negative infinity inside the mem-eff kernel. I'm down to accept, but I think this gives me an uneasy feeling and at least we should maybe have tests on what this is trying to change

Reply:
Yeah, this magic number stuff doesn't feel great to me either.